src/jit/gentree.cpp

```diff
@@ -13044,6 +13116,38 @@ bool GenTree::DefinesLocalAddr(Compiler* comp, unsigned width, GenTreeLclVarComm
     return false;
 }
 
+//------------------------------------------------------------------------
+// IsLocalExpr: Determine if this is a LclVarCommon node
+//
```

Comment:
This shodul probably read:
Determine if this is a LclVarCommon node and return some additional info in the two out parameters.